docs/pre-req.rst

```diff
@@ -1,15 +1,15 @@
-Before You Install
-==================
+Pre-requisites
+==============
 
-Before you install the F5 ACI ServiceCenter, read the following information:
+Pre-requisites for the F5 ACI ServiceCenter:
 
-- To enable and use the L4-L7 Configuration tab, use the BIG-IP Configuration utility to install the f5-appsvcs package version 3.19.1+ on the BIG-IP.
-- You must use APIC GUI to add a logical device. The F5 ACI ServiceCenter does not provide this functionality.
-- For each BIG-IP device added to the F5 ACI ServiceCenter, only one BIG-IP user can log in at a time.
-- Multiple APIC admin users will see the same view of the app (the same list of BIG-IP devices with the same application data).
+- F5 ACI ServiceCenter application should be a part of the default security domain for proper functioning.
+- Cisco APIC user accessing the App must have admin privileges to the APIC.
 - All BIG-IP devices added to the F5 ACI ServiceCenter should have https support.
+- To enable and use the L4-L7 Configuration tab, use the BIG-IP Configuration utility to install the f5-appsvcs package version 3.19.1+ on the BIG-IP device. Follow the installation steps from https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/installation.html
+- To enable and use basic subtab of the L4-L7 Configuration tab ,use the BIG-IP Configuration utility to install f5-appsvcs-templates package version 1.8.1+ on the BIG-IP device. Follow the installation steps from https://clouddocs.f5.com/products/extensions/f5-appsvcs-templates/latest/userguide/install-uninstall.html
+- To view the Virtual Server statistics on the Visibility VIP Dashboard, you must install the Telemetry Streaming plugin version 1.17 or higher on the BIG-IP device. Follow the installation steps and default pull consumer configuration details from https://clouddocs.f5.com/f5-aci-servicecenter/latest/navigate.html#configure-telemetry 
+- To enable and view discovered devices in F5 ACI ServicCenter , enable LLDP protocol on both BIG-IP as well as on APIC. Make sure management IP of the BIG-IP is present in the LLDP attributes on BIG-IP(Network->Interfaces->Interfaces List-><interface_id>).
```

Comment:
..ACI ServiceCenter, enable the LLDP protocol on both the BIG-IP... (note comma/spacing, and typo in servicecenter)